composer.json

```diff
@@ -37,7 +37,11 @@
             "find vendor -name doc -type d -exec rm -r {} +",
             "find vendor -name docs -type d -exec rm -r {} +",
             "find vendor -name tests -type d -exec rm -r {} +",
-            "find vendor -name build -type d -exec rm -r {} +"
+            "find vendor -name build -type d -exec rm -r {} +",
+            "mkdir -p vendor/sqlftw/sqlftw/tests",
+            "mkdir -p vendor/sqlftw/sqlftw/build",
+            "mkdir -p vendor/sqlftw/sqlftw/build",
+            "mkdir -p vendor/dogma/dogma/tests"
```

Comment:
we need to re-create empty folders because of upstream libs have this folders in the `autoload` and not `autoload-dev` classmap path

- https://github.com/SQLFTW/sqlftw/pull/24